app/controllers/ips_controller.rb

```diff
@@ -4,7 +4,7 @@ def new
   end
 
   def create
-    @ip = current_organisation.ips.new(ip_params)
+    @ip = current_organisation.locations.first.ips.new(ip_params)
```

Comment:
hack

Reply:
is it worth explicitly calling out what this location is at the moment?

```
default_location = current_organisation.locations.first
@ip = default_location.ips.new(ip_params)
```